pyEpiabm/pyEpiabm/utility/distance_metrics.py

```diff
@@ -101,3 +102,29 @@ def dist_periodic(loc1: typing.Tuple[int, int],
             if diff[index] > 0.5*scales[index]:
                 diff[index] = scales[index] - diff[index]
         return np.linalg.norm(diff)
+
+    def minimum_between_cells(cell1, cell2):
```

Comment:
At the moment this will result in certain infection, as we don't have a spatial microcell distribution, so the distance between microcells will be the same as the distance between cells, and so the ratio of spatial kernels will be unity. Ideally we should implement spatial distribution of microcells, but at the very least log a warning here if the minimum distance is the same as the distance between cells, so that the user is aware that microcell locations probably aren't set, and you may be getting certain infection on the spatial sweep

Reply:
After discussion will set microcells in the toy population config on the next pull request

Reply:
Have now implemented